pkg/asset/tls/variables.go

```diff
@@ -0,0 +1,228 @@
+package tls
+
+import (
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/openshift/installer/pkg/types"
+)
+
+const (
+	validityTenYears      = time.Hour * 24 * 365 * 10
+	validityThirtyMinutes = time.Minute * 30
+)
+
+func init() {
+	RootCA := &CertKey{
+		IsRoot:       true,
+		Subject:      pkix.Name{CommonName: "root-ca", OrganizationalUnit: []string{"openshift"}},
+		KeyUsages:    x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
+		Validity:     validityTenYears,
+		KeyFileName:  "root-ca.key",
+		CertFileName: "root-ca.crt",
+
+		IsCA: true,
+	}
+
+	KubeCA := &CertKey{
+		Subject:      pkix.Name{CommonName: "kube-ca", OrganizationalUnit: []string{"bootkube"}},
```

Comment:
How important is it to vary `OrganizationalUnit`?  I'd expect that `CommonName` is sufficient context and we can just hard-code `[]string{"openshift"}` for this in your cert-`Asset` method (instead of having it configurable via a cert-`Asset`-struct property).

Reply:
:+1:

Reply:
Hmm, there's also `Organization` field, I am not sure what's the best way of dealing with all those, so maybe just leave as is for now, and we can refactor later?